test/DetailedResultsBuilder.test.ts

```diff
@@ -15,6 +15,61 @@ const cvMeasure = getJSONFixture('measure/cv-measure.json') as MeasureWithGroup;
 const simpleMeasureGroup = simpleMeasure.group[0];
 const cvMeasureGroup = cvMeasure.group[0];
 
+const measure: fhir4.Measure = {
```

Comment:
I would extract this measure out to the `fixtures/measure` folder of the test directory and pull it in using the `getJSONFixture` helper function